docs/en_US/Compressor/Framework.md

```diff
@@ -142,3 +148,125 @@ self.pruner.remove_activation_collector(collector_id)
 
```

Comment:
A basic pruner -> A basic`pruner`

Reply:
which one?

Reply:
It's in the old version, at Line83 `A basic pruner looks likes this:`

Reply:
fixed